react/locales/fr.json

```
@@ -34,9 +34,11 @@
   "addresses.addAddress": "Ajouter addresse",
   "addresses.saveAddress": "Sauvegarder addresse",
   "addresses.deleteAddress": "Supprimer addresse",
+  "addresses.notFound": "Vous n'avez aucune adresse enregistré!",
```

Exclamation points take a space before in french.
```suggestion
  "addresses.notFound": "Vous n'avez aucune adresse enregistré !",
```

---

D'accord 👍